src/cantypes.hpp

```diff
@@ -32,20 +44,24 @@ struct CANsignal {
 struct CANmessage {
     // Constructor required for vs2015 to be able to use initializer_list
     CANmessage(std::uint32_t _id, const std::string& _name = "",
-        std::uint32_t _dlc = 0, const std::string& _ecu = "")
+        std::uint32_t _dlc = 0, const std::vector<std::string>& _ecus = { },
+        boost::optional<std::uint32_t> _cycleTime = boost::none,
```

Comment:
is cycle time supposed to be 32bit or 64bit? In CanMessages object i used 63 bit unsigned ints for this.. but would be nice to use the same datatype everywhere.

Reply:
I will strongly assume 32-bit, because even 64-bit values are written as 32-bit numbers in the DBC file (as we saw when dealing with values). Anyway, this is milliseconds -- and I don't anticipate anyone using a cycle time of more than 4294967295ms = ~7.1 **weeks** :innocent:

Reply:
Updated my model and protobuf files for this.